scope.ml

```diff
@@ -24,24 +36,29 @@ let infer ({formals; instrs} : analysis_input) : inferred_scope array =
 
   let infer_scope instructions =
     let merge pc cur incom =
-      if not (VarSet.equal cur.info incom.info)
+      if not (DeclSet.equal cur.info incom.info)
       then raise (IncompatibleScope (cur, incom, pc))
       else if PcSet.equal cur.sources incom.sources then None
       else Some { info = cur.info; sources = PcSet.union cur.sources incom.sources }
     in
     let update pc cur =
       let instr = instructions.(pc) in
-      let added = Instr.declared_vars instr in
+      let declared = Instr.declared_vars instr in
```

Comment:
Could you use different names for `declared` and `declared'`? The names are too similar for me.